optuna/samplers/_grid.py

```diff
@@ -97,9 +98,12 @@ def objective(trial):
         search_space:
             A dictionary whose key and value are a parameter name and the corresponding candidates
             of values, respectively.
+        seed: Seed for random number generator.
```

Comment:
I think the randomness of `GridSampler` is not intuitive. Could you make clear that this argument specifies the evaluation order of trials?

Reply:
Thank you for comments!
I revised it.